changelog/17078.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+agent/consistency: Fix issue where X-Vault-Index header from auto-auth was not being passed to API proxy.
```

Comment:
Can this be phrased in a way that will be more meaningful to a user, i.e. more in terms of user-visible behaviour?  Why is this is an issue the reader should care about?